rero_ils/modules/items/templates/rero_ils/detailed_view_items.html

```diff
@@ -51,7 +55,10 @@ <h1 class="col-sm-11 col-md-11">{{ _('Barcode') }} {{ record.barcode }}</h1>
             library.name)
           }}
           {% if record.status %}
-            {{ dl(_('Status'), _(record.status)) }}
+            {{ dl(
+              _('Availability'),
+              '<i class="fa fa-circle text-' + availability + '"></i> ' + record|item_availability_text)
```

Comment:
you may remove lines 37-40 and  replace  `availability` by :
```
{{ 'success' if record.available else 'danger' }}
```

Reply:
@BadrAly I did it L.37 because L.60 didn't accept `{{` and `}}` and without them it didn't use the end of the line which break the page.

Reply:
ok